amqVivaceShortcuts.user.js

```diff
@@ -1,9 +1,9 @@
 // ==UserScript==
-// @name         AMQ Vivace! Shortcuts
+// @name         AMQ Vivace! Optimized Shortcuts
 // @namespace    http://tampermonkey.net/
 // @version      1.6
```

Comment:
Should bump the version
```suggestion
// @version      1.7
```